sklearn/plot/_gridsearch_results.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
+from sklearn.plot import plot_heatmap
+
+
+def _plot_1D_results(cv_results, params, ax, xlabel, ylabel, title,
+                     fmt, xtickrotation):
+    import matplotlib.pyplot as plt
+
+    param = params[0]
+    param_range = sorted(cv_results['param_%s' % param])
+    train_scores_mean = cv_results['mean_train_score']
+    train_scores_std = cv_results['std_train_score']
+    test_scores_mean = cv_results['mean_test_score']
+    test_scores_std = cv_results['std_test_score']
+
+    lw = 2
+    x_vales = range(len(param_range))
+    plt.plot(x_vales, train_scores_mean,
```

Comment:
That should probably be done on the `ax` object, and not using the functional API of matplotlib but the OO one.